### ISSUE TITLE
Problem with patterns containing double quotes

### ISSUE THREAD

**Issue by rrthomas** [kopoli/helm-grepint#16]
For example, if I try to search for `""`, I get the error:

Error running timer: (error "Candidates function ‘helm-grepint-grep-process’ should run a process")

**Comment by kopoli** [comment]
Thanks for reporting this ! (And finding so many bugs so far!)

I could reproduce this with both `git-grep` and `ag` (therefore it most likely happens before running the command). With a quick enabling-of-debugging there is at least a problem with quote mangling just before the command is run.

I'll look further into this.

**Comment by kopoli** [comment]
The problem seemed to be that the double-quote replacement was too fancy for it's own good. The `literal` option needed to be enabled to dumb it down to suit this use case.

I don't get the error any more with the changes in #17.

**Comment by rrthomas** [comment]
Many thanks! Works perfectly for me.
